data-type.md

```diff
@@ -0,0 +1,35 @@
+# ডাটা টাইপ
+
+| ডাটা টাইপ |  বর্ননা | ডাটা সাইজ |
+| -- | -- | -- |
+|char(n)|Fixed width character string. Maximum 8,000 characters|Defined width|
```

Comment:
please update this pages formatting. you can use http://www.tablesgenerator.com/markdown_tables